client/my-sites/theme/main.jsx

```diff
@@ -346,21 +347,35 @@ const ThemeSheet = React.createClass( {
 	},
 
 	renderSupportTab() {
-		if ( this.props.isCurrentUserPaid ) {
+		const { isCurrentUserPaid, isJetpack, forumUrl, isWpcomTheme, isLoggedIn } = this.props;
+		let buttonCount = 1;
+
+		if ( isLoggedIn ) {
 			return (
 				<div>
-					{ this.renderContactUsCard( true ) }
-					{ this.renderThemeForumCard() }
-					{ this.renderCssSupportCard() }
+					{ isCurrentUserPaid && ! isJetpack &&
+						this.renderSupportContactUsCard( buttonCount++ ) }
+					{ forumUrl && isWpcomTheme &&
```

Comment:
That's... correct in the sense that we aren't really able to provide support. 

Maybe we can just see if there's a URL? Yes it might make sense, as it should point elsewhere if it's not a .com theme...